RubeesEat/Startup.cs

```diff
@@ -38,8 +38,8 @@ public void ConfigureServicesEAT(IServiceCollection services)
         var billRepository = new InMemoryBillRepository();
         billRepository.Add(new Bill(Guid.NewGuid(), new DateTime(2024, 9, 9), "Mittagessen auf Patrick sein Nacken",
         [
-            new EntryLine(persons[1], 1_000_000),
-            new EntryLine(persons[2], -500_000),
+            new EntryLine(persons[0], (decimal)1_000_000.50),
+            new EntryLine(persons[2], (decimal)-500_000.50),

```

Comment:
In this case I would prefer adding a type suffix instead of doing a cast:

```suggestion
            new EntryLine(persons[0], 1_000_000.50m),
            new EntryLine(persons[2], 500_000.50m),

```

`m` stands for `decimal` (**m**oney)